guide/faq.md

```diff
@@ -39,19 +37,6 @@ not modify built-in APIs) if it uses a browser API that is not widely supported
 However, if you are using the ExpandCollpase component, we recommend the use of the Set object in [polyfill.io](https://polyfill.io/v2/docs/) (see
 [starter kit example][starter-kit-polyfill]) for rendering the ExpandCollapse in older browsers.
 
-## What global styles does TDS set on the page?
-
-The majority of TDS styles are component styles that have been generated by our build tooling to prevent conflicts with application styles. However, there are a small amount of page level styles to establish a common baseline.
-
-Specifically, the `tds.css` stylesheet contains the following page level styles:
-
-* **reset**: to standardize default styling of HTML5 elements across browsers, using the popular [Eric Meyer Reset](http://meyerweb.com/eric/tools/css/reset/)
-* **font-face declarations**: to load the TELUS font files
-* **border-box**: to make it easier to size elements by telling the browser to account for any border and padding in the value you specify for width and height
-* **default body styles**: to establish the base font size and colour
-
-Please do not override these global styles. Doing so will likely produce non-deterministic behaviour and UI defects.
-
```

Comment:
Hmmmm, I'm starting to feel bad about the back and forth.... :(

I did not mean to imply that we ditch the "What global styles..." section altogether. I think it is still a common question, and something users may wonder about. The context of scoping styles is important.

What I meant is to leave this question in the FAQ, keep the part about "The majority of TDS styles are component styles...", and then include a part that says that the only package that DOES set global styles is the CSS Reset package. And include a link to more documentation about that package, either to npm or the component docs.

Does that make sense? Happy to chat about it.